docs/source/azure/policy/resources/cdn.rst

```diff
@@ -0,0 +1,45 @@
+.. _azure_cdn:
+
+CDN
+===
+
+Filters
+-------
+- Standard Value Filter (see :ref:`filters`)
+      - Model: `Profile <https://docs.microsoft.com/en-us/python/api/azure-mgmt-cdn/azure.mgmt.cdn.models.profile?view=azure-python>`_
+- ARM Resource Filters (see :ref:`azure_genericarmfilter`)
+    - Tag Filter - Filter on tag presence and/or values
+    - Marked-For-Op Filter - Filter on tag that indicates a scheduled operation for a resource
+
+Actions
+-------
+- ARM Resource Actions (see :ref:`azure_genericarmaction`)
+
+Example Policies
+----------------
+
+This set of policies will mark all CDNs for deletion in 7 days that have 'test' in name (ignore case),
+and then perform the delete operation on those ready for deletion.
+
+.. code-block:: yaml
+
+    policies:
```

Comment:
This is sort of a weird example policy to use everywhere :)  The mark for op stuff is a little complicated/confusing and not something people can use to easily try out the resource.  Just a value filter on some semi-relevant field to the resource is fine with no action (profile might let you do something like filter on sku = `Standard_Verizon` etc)